src/service/resource.js

```diff
@@ -901,7 +901,11 @@ export class Resource {
    * @return {boolean}
    */
   isInViewport() {
-    return this.element.isInViewport();
+    const isInViewport = this.element.isInViewport();
+    if (isInViewport) {
```

Comment:
return this.element.isInViewport() ? this.resolveRenderOutsideViewport_() || true : false;

Reply:
![image](https://user-images.githubusercontent.com/4276992/40723609-4104afde-63ed-11e8-925e-e79a540f80ce.png)

Reply:
(don't actually do this)

Reply:
Let's not golf.

Reply:
agreed :D